src/main.c

```diff
@@ -804,6 +854,7 @@ int main(int argc, char *argv[])
     showmsg(tlfversion);
     showmsg("");
 
+    total = 0;
```

Comment:
Explicit is better than implicit :). In readcalls(), the total initialization is only after a condition, but the readqtccalls() also initializes the total variable. I think, now it's better to keep this.